task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +594,43 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    const win = [[0, 1, 2], // Check first Row.  
+              [3, 4, 5], // Check second Row  
+              [6, 7, 8], // Check third Row  
+              [0, 3, 6], // Check first column  
+              [1, 4, 7], // Check second Column  
+              [2, 5, 8], // Check third Column  
+              [0, 4, 8], // Check first Diagonal  
+              [2, 4, 6]]; // Check second Diagonal
+               
+              for (let i = 0; i < 3 ; i++){
+              if(position[i].length < 3){
+                for(let j = 0; j < 3 - position[i].length; j++)
+                    position[i].push(undefined);
+              };
+            };
+              let newBoard = [].concat.apply([], position);
+
+              const isCWin = function(position, char) {
```

Comment:
JFYI: best practice - create new functions in global scope. If you create function in another function it can implicitly lock the scope of function and it can lead to memory leak.